### ISSUE TITLE
Why does the objectness_flag set to False during training?

### ISSUE THREAD

**Issue by seanzhuh** [filaPro/oneformer3d#106]
Hi, why not let the model predict objectness score during training? It seems that [mask logits who are positive are summed up to form the objectness score](https://github.com/filaPro/oneformer3d/blob/main/oneformer3d/oneformer3d.py#L86) instead of [having a separate branch to predict the objectness score](https://github.com/filaPro/oneformer3d/blob/main/oneformer3d/oneformer3d.py#L69)

**Comment by filaPro** [comment]
I think we just borrowed this trick from SPFormer or Mask3D.
